samples/src/main/java/com/example/bigquery/LoadParquet.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_load_table_gcs_parquet]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.FormatOptions;
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.LoadJobConfiguration;
+import com.google.cloud.bigquery.TableId;
+import java.math.BigInteger;
+
+public class LoadParquet {
+
+  public static void runLoadParquet() {
+    // TODO(developer): Replace these variables before running the sample.
+    String datasetName = "MY_DATASET_NAME";
+    loadParquet(datasetName);
+  }
+
+  public static void loadParquet(String datasetName) {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      String sourceUri = "gs://cloud-samples-data/bigquery/us-states/us-states.parquet";
+      TableId tableId = TableId.of(datasetName, "us_states");
+
+      // For more information on LoadJobConfiguration see:
+      // https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/bigquery/LoadJobConfiguration.Builder.html
```

Comment:
nit: I'm not sure a link to java-doc is super useful, but possibly linking to FormatOptions might be more useful: https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/bigquery/FormatOptions.html (so users can see what FormatOptions are available) 

But this sample is specific to the parquet format, so I don't think it's really required.